content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -11,7 +11,7 @@ weight: 10
 
 CronJobs was promoted to general availability in Kubernetes v1.21. If you are using an older version of
 Kubernetes, please refer to the documentation for the version of Kubernetes that you are using,
-so that you see accurate information. Older Kubernetes versions do not support the `batch/v1` CronJob API.
```

Comment:
The original wording looks like idiomatic English to me.

Reply:
However, for v1.25 we should actually remove this initial sentence. I filed issue https://github.com/kubernetes/website/issues/34610 for that.